contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -143,6 +146,12 @@ message ApiAuthenticationConfig {
   bool force_disable = 1;
 }
 
+// Server config for API Authorization via Firebase Rules
+message ApiCheckSecurityRulesConfig {
+  // Allows to disable the API authorization
+  bool force_disable = 1;
+}
```

Comment:
I don't want to disable this by default via server config and there is a reason for this. By not disabling the feature by default I am able to apply some existing auth based t tests. For example, we have checks to ensure that if auth is disabled, then we don't do any firebase rules checks too. For all those tests, since the code is not disabled by default, we are able to exercise the existing t tests without any more tests that do the same thing.